webpack.config.js

```diff
@@ -9,7 +11,8 @@ module.exports = {
   resolve: {
     alias: {
       "devtools": "ff-devtools-libs",
-      "sdk": "ff-devtools-libs/sdk"
+      "sdk": "ff-devtools-libs/sdk",
+      "themes": environment.geckoPath + "devtools/client/themes"
```

Comment:
It seems silly to add gecko as a dependency just for some css, but we'll probably need it for something soon and our target audience has gecko... What do you think?